rules/do-you-securely-implement-and-handle-jwt-tokens/rule.md

```diff
@@ -0,0 +1,74 @@
+---
+type: rule
+title: Do you securely implement and handle JWT Tokens?
```

Comment:
The T in JWT stands for token so "JWT Tokens" is redundant. I would suggest change to "Do you securely implement and handle Json Web Tokens (JWTs)?

Reply:
As per our conversation the rule was changed entirely